### PR TITLE
Create Release40update

### PR DESCRIPTION
**Pull request by TCP1231**

### CODE REVIEW

**Review by TCP1231 — COMMENTED** [review]
updated as discussed

**Review by vladvino — APPROVED** [review]
Thank you!

File: Release40update [hunk 1]
```diff
@@ -0,0 +1,27 @@
+### Highlights
+ 
+- We're excited to announce availability of the Public Preview of our latest Azure API Management pricing tiers: [Basic v2 and Standard v2](https://learn.microsoft.com/azure/api-management/v2-service-tiers-overview).
+- API Management is now Generally Available in the Israel Central and Italy North Azure regions.
+ 
+### New features, fixes and improvements
+ 
+- You can now specify timeout in milliseconds in the the `forward-request` policy via the new `timeout-ms` attribute.
```
Comment (simonkurtz-MSFT) [inline Release40update:8]: @vladvino, double `the` here

File: Release40update [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+- We changed `validate-jwt` policy to properly handle `Bearer` and other authorization token schemas correctly in custom headers.
+- We fixed an issue where a large request log entry blocked Azure monitor log stream.
+- We fixed an issue where the `emit-metric` policy could fail the request when no diagnostic entries are configured at any of the scopes.
```
Comment (simonkurtz-MSFT) [inline Release40update:11]: @vladvino, I believe 
`when no diagnostic entries are configured`
should be
`when no diagnostic entries were configured` 
as this is no longer an issue now after the fix.

File: Release40update [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+- We fixed an issue where a request to a GraphQL API with empty GraphQL resolver policy fails the request.
```
Comment (simonkurtz-MSFT) [inline Release40update:12]: @vladvino, I believe `fails` should be `failed` as the issue has been fixed.

File: Release40update [hunk 4]
```diff
@@ -0,0 +1,27 @@
+### Highlights
+ 
+- We're excited to announce availability of the Public Preview of our latest Azure API Management pricing tiers: [Basic v2 and Standard v2](https://learn.microsoft.com/azure/api-management/v2-service-tiers-overview).
+- API Management is now Generally Available in the Israel Central and Italy North Azure regions.
+ 
+### New features, fixes and improvements
+ 
+- You can now specify timeout in milliseconds in the the `forward-request` policy via the new `timeout-ms` attribute.
+- We changed `validate-jwt` policy to properly handle `Bearer` and other authorization token schemas correctly in custom headers.
```
Comment (tomkerkhove) [inline Release40update:9]: We should remove this, I deliberately did not share this as  there was a regression =)

Can you please remove this?

**Review by tomkerkhove — CHANGES_REQUESTED** [review]
Added a request to remove and believe we should make these changes in https://github.com/Azure/API-Management/blob/main/changelogs/api-management-service.md instead

**Pull request closed without merging**